hasher-matcher-actioner/hmalib/lambdas/api/matches.py

```diff
@@ -118,7 +118,19 @@ def matches() -> MatchSummariesResponse:
         """
         Returns all, or a filtered list of matches.
         """
-        records = PDQMatchRecord.get_from_time_range(dynamodb_table)
+        signal_q = bottle.request.query.signal_q or None
```

Comment:
Looks good, my only concern is documentation, the endpoint is now complex enough the doc string should be more detailed and ideally include example usages (if not an API README)